x-pack/plugins/observability_solution/infra/common/http_api/metrics_api.ts

```diff
@@ -10,16 +10,11 @@ import * as rt from 'io-ts';
 import { MetricsUIAggregationRT } from '@kbn/metrics-data-access-plugin/common';
 import { afterKeyObjectRT } from './metrics_explorer';
 
-export const MetricsAPITimerangeRT = rt.intersection([
-  rt.type({
-    from: rt.number,
-    to: rt.number,
-    interval: rt.string,
-  }),
-  rt.partial({
-    timeFieldName: rt.string,
-  }),
-]);
+export const MetricsAPITimerangeRT = rt.type({
+  from: rt.number,
+  to: rt.number,
+  interval: rt.string,
+});
```

Comment:
Can we remove this and only use this one https://github.com/elastic/kibana/pull/178125/files#diff-1cc197d4b7c01f256aa0e32761b8396567e92e962b5b3241756e4671bbfbc7b8R77?

Reply:
Fixed in [d0d46a8](https://github.com/elastic/kibana/pull/178125/commits/d0d46a86ecb952d0120b1c7a55e6041542be7104).